openstack/02b-MAChigh/schedule.c

```diff
@@ -521,6 +529,49 @@ slotOffset_t schedule_getNextActiveSlotOffset() {
 }
 
 /**
+\brief return slotOffset of closest active slot

```

Comment:
In the comments, add:
- when this function is called and by which module
- what does "closest" mean? Only in the future, or also in the future AND past.